src/components/Newsfeed.vue

```diff
@@ -1,7 +1,7 @@
 <!-- eslint-disable vue/no-v-html -->
 <template>
 	<div class="newsFeed">
-		<div v-for="message in news" :key="message.key" class="message-box">
```

Comment:
This is causing a few warnings in the console because subjects from some messages are the same. We can start using the activity_id property as a key, since every event from the activity app has a different one and add this property to the hard coded data